csvkit/utilities/csvjson.py

```diff
@@ -110,39 +144,48 @@ def dump_json(data, newline=False):
                 feature['type'] = 'Feature'
                 properties = OrderedDict()
                 geoid = None
+                geo_type = 'Point'
                 lat = None
                 lon = None
+                coords = None
 
                 for i, c in enumerate(row):
                     if i == lat_column:
+                        if not c:
```

Comment:
Yes, correct.
However, this only occurs when the type is not `Point`, and fixing this exception only causes the tool to generate a .geojson which is broken in other ways.  Fixing only the exception doesnt make the tool better; in fact, I would say that the exception is a good thing, as it prevents people from believing that their input was correctly parsed.

But, happy to split to separate it to a separate commit - other improvements needed anyway.